README.md

```diff
@@ -1,114 +1,112 @@
 
 And boom! You have successfully installed QuestCraft on your VR headset! To play, press the **Play** button and wait for it to load; depending on the headset's age, it might take a while.
 
-## Supported Renderers (Take note when installing mods!)
+**NOTE:** You may encounter crashes when downloading your selected MC version, mainly on v53 and later. This happens because Meta can't fix memory leaks for some reason. Don't fret; launch the app, press play, and the download will continue from where it left off before the crash. If it continues, please keep trying. If, after 4 attempts, you still are not loaded in, please visit the support channel of the QuestCraft Discord (discord.gg/questcraft)
 
-- ### Zink
-**[Zink](https://docs.mesa3d.org/drivers/zink.html)**, works with many mods, at the cost of performance. (which is mostly fixed with the performance mods below)
-- ### Vulkan
-**[Vulkan](https://www.vulkan.org/)**, while very fast, it may only works with a small amount of mods. For a list, please see the pinned discussion in #discussion on the **[VulkanMod Server](https://discord.gg/FVXg7AYR2Q)**. VulkanMod is not affiliated with us, so please don't ask and spam the devs questions related to QuestCraft! 
-*Vulkan is not yet out for QuestCraft, please wait!*
+### Please do not enable seated mode in the in-game VR settings!
 
 ## Included Mods 
 
-**Did you experience a mod-related crash when loading MC? You can do 1 of these solutions to find the cause and fix it!**
-
-**Easy Way**                                                                                                                                                                                                                                                                                  
-Join the QuestCraft [Discord Server](https://discord.gg/questcraft) and go to #bot-commands. Type /logs and upload your log file (which can be retrieved from Android/data/com.qcxr.qcxr using a file viewer e.g Windows Explorer). The QuestCraft Bot, Crafty, will give you feedback and help you fix any issues.
+**Did you experience a mod-related crash when loading MC? It's (kind of) simple to fix it!**
+                                                                                                                                                                                                                                                                                  
+Join the QuestCraft Discord Server at **discord.gg/QuestCraft** and go to #bot-commands. Type /logs and upload your log file (which can be retrieved from Android/data/com.qcxr.qcxr using a file viewer (e.g. **Windows Explorer** or **[Android File Transfer for macOS](https://www.android.com/filetransfer/)**). The QuestCraft Bot, CraftyVR, will give you ways to fix the crash. If not, you can scroll to the bottom of the retrieved log in a text editor and check the Fabric error message for causes and possible solutions. 
```

Comment:
uploading logs is now a lot easier, there's a button for it in the launcher.
the command was renamed to analyze, it also doesnt really help crashes.

Reply:
I'll simply remove this part for now.